gax-grpc/src/test/java/com/google/api/gax/grpc/GrpcOperationCallableImplTest.java

```diff
@@ -843,4 +858,59 @@ private void mockResponse(ManagedChannel channel, Status.Code statusCode, Object
   private UnaryCallable<Integer, Operation> createDirectCallable() {
     return new GrpcDirectCallable<>(FakeMethodDescriptor.<Integer, Operation>create());
   }
+
+  private void assertExceptionType(GrpcStatusCode code, Throwable exception) {
```

Comment:
The name of this helper doesn't quite convey a full concept. Maybe: `assertExceptionMatchesCode`

Reply:
Done